### PR TITLE
Release 1.1 fixes

### PR DESCRIPTION
**Pull request by pconesa**

### CODE REVIEW

File: pyworkflow/apps/pw_chimera_client.py [hunk 1]
```diff
@@ -24,13 +24,12 @@ def main():
     projectorParser.add_argument('--projectionSize', help='Projection window dimensions', type=int)
     projectorParser.add_argument('--paddingFactor', default=1, type=float, help='Projection padding factor')
     projectorParser.add_argument('--maxFreq', default=0.5, type=float, help='Projection spline degree')
-    projectorParser.add_argument('--splineDegree', default='BSPLINE2',
-                        choices=['NEAREST', 'LINEAR', 'BSPLINE2', 'BSPLINE3', 'BSPLINE4'], help='projection window dimensions')
+    projectorParser.add_argument('--splineDegree', default='BSPLINE3',
```
Comment (delarosatrevin) [inline pyworkflow/apps/pw_chimera_client.py]: Do we really need this interpolation as default for the projection?

File: pyworkflow/apps/pw_chimera_client.py [hunk 2]
```diff
@@ -24,13 +24,12 @@ def main():
...
+                        choices=['NEAREST', 'LINEAR', 'BSPLINE3'], help='projection window dimensions')
```
Comment (delarosatrevin) [inline pyworkflow/apps/pw_chimera_client.py]: Moreover, the help message does not seems very accurate.

File: pyworkflow/em/viewer.py [hunk 3]
```diff
@@ -610,7 +610,7 @@ def __init__(self, volfile, **kwargs):
         self.size = size if size else defaultSize
         paddingFactor = self.kwargs.get('paddingFactor', 1)
         maxFreq = self.kwargs.get('maxFreq', 0.5)
-        splineDegree = self.kwargs.get('splineDegree', 2)
+        splineDegree = self.kwargs.get('splineDegree', 3)
```
Comment (delarosatrevin) [inline pyworkflow/em/viewer.py]: The same here...for visualization do we need this higher spline order? which makes computations slower.

File: pyworkflow/tests/em/protocols/test_protocols_mixed_movies.py [hunk 4]
```diff
@@ -91,7 +91,8 @@ def _compareMovies(self, micSet1, micSet2):
         for mic1, mic2 in izip(micSet1, micSet2):
             img1.read(mic1.getFileName())
             img2.read(mic2.getFileName())
-            self.assertTrue(img1.equal(img2, 0.001))
+            print("Comparing %s vs %s"%(mic1.getFileName(),mic2.getFileName()))
+            self.assertTrue(img1.equal(img2, 20))
```
Comment (delarosatrevin) [inline pyworkflow/tests/em/protocols/test_protocols_mixed_movies.py]: Having this big value of "different" seems to much for me. I think there could be a hidden problem somewhere else.

File: software/em/xmipp/libraries/bindings/python/python_fourierprojector.cpp [hunk 5]
```diff
@@ -99,7 +99,7 @@
          double padding_factor, max_freq, spline_degree;
          padding_factor = 2;
          max_freq = 0.5;
-         spline_degree = 2;
+         spline_degree = 3;
```
Comment (delarosatrevin) [inline software/em/xmipp/libraries/bindings/python/python_fourierprojector.cpp]: Same here....

File: pyworkflow/apps/pw_chimera_client.py (hunk 1 above)
Comment (cossorzano) [inline pyworkflow/apps/pw_chimera_client.py]: Yes, actually Bspline2 and Bspline4 are not implemented in the C side

File: pyworkflow/apps/pw_chimera_client.py (hunk 2 above)
Comment (cossorzano) [inline pyworkflow/apps/pw_chimera_client.py]: I did not see the help, it stayed as it was, and it is certainly wrong

**Pull request merged**